### PR TITLE
Expose stderr and stdout

### LINKED ISSUE #13: [Feature] Expose stdout and stderr
:wave: @kevin940726 

Would you be open to tweak `TestingConsoleInstance` so that it exposes `stderr` and `stdout` string properties?

`stdout` would be valued by invocations leveraging `log` and `info`, and `stderr` by `warn` and `error`.

### PR DESCRIPTION
**Pull request by nulltoken**
Fix #13

Also:
- Fix export paths (to avoid VSCode warnings)
- Bump dependencies

### CODE REVIEW

**Comment by nulltoken** [comment]
@kevin940726 Here's a first WIP PR to get your early feedback

**Comment by nulltoken** [comment]
@kevin940726 FWIW, in #13 you mentioned _"I don't think exposing them to Node environment only would be a bad idea."_ 

The _"only"_ part of this sentence confuses me a bit. Could you please elaborate a bit about what you'd like to see?

**Comment by kevin940726** [comment]
> The "only" part of this sentence confuses me a bit. Could you please elaborate a bit about what you'd like to see?

Right, It wasn't actually mean "only", just that this API would only make sense in Node.js environment, which I think is fine.

**Review by kevin940726 — APPROVED** [review]
Code looks good! Now we need some documentation in README 😉.

**Comment by nulltoken** [comment]
> Code looks good! Now we need some documentation in README 😉.

@kevin940726 Done.

File: README.md [hunk 1]
```diff
@@ -125,7 +125,7 @@ test('testing console.log 2', () => {
 Every log have a corresponding **logging level**, you can access each level's log via `getLog().levels`, or access all of them in a list with `getLog().logs`.
 
 ```js
-console.info('This is %s level', 'log');
+console.log('This is %s level', 'log');
```
Comment (kevin940726) [inline README.md:128]: Ahh, good catch!

**Review by kevin940726 — APPROVED** [review]

**Pull request merged**